libarchive/PKGBUILD

```diff
@@ -36,7 +36,11 @@ build() {
     --enable-shared \
     --enable-static \
     --without-libiconv-prefix \
-    --without-xml2
+    --without-xml2 \
+    --without-cng
```

Comment:
Is there a downside to doing this?

Reply:
According to https://github.com/libarchive/libarchive/blob/a163558c388bb2ccedef683e8031c11193e6875c/CMakeLists.txt#L210 libarchive would no longer be able to handle encrypted archives.

Reply:
I think this only applies to windows zip archives. Other formats are handled by the respective delegate library (bz2, lzma, etc).

I think there is little need to support encrypted zip files in msys2 runtime (of course they still work in the native `mingw-w64-libarchive`)